create_package/action.yaml

```diff
@@ -0,0 +1,49 @@
+name: 'Create development package'
+description: 'Creates package with dev version and uploads it to python index'
+
+branding:
+  icon: 'battery-charging'
+  color: 'green'
+
+inputs:
+  pypi_repo_url:
+    description: URL of python package index (for custom packages)
+    required: true
+
+  pypi_token_username:
+    description: Username for authentication at python package index (for custom packages)
+    required: true
+
+  pypi_token:
+    description: Token for authentication at python package index (for custom packages)
+    required: true
+
+  package_name:
+    description: Name of package to create. This should match name of root project directory
+    required: true
+
+outputs:
+  version:
+    description: "Version"
+    value: ${{ steps.package-generator.outputs.version }}
+
+runs:
```

Comment:
Couple things:

- Can this action check that a PR/issue is still open? We don't want packages to be published from a closed PR/issue.
- For public packages - if repo url is not provided, we should default to test.pypi.org
- Does CUSTOM_REPO thing work with pypi?

Reply:
>Can this action check that a PR/issue is still open? We don't want packages to be published from a closed PR/issue.

If the job is defined as in Usage section, yes, the job will not run if PR is closed

>For public packages - if repo url is not provided, we should default to test.pypi.org

I think, it's better to not use default values, since it's error prone.